packages/block-editor/src/hooks/anchor.js

```diff
@@ -71,7 +71,18 @@ export const withInspectorControl = createHigherOrderComponent( ( BlockEdit ) =>
 					<InspectorAdvancedControls>
 						<TextControl
 							label={ __( 'HTML Anchor' ) }
-							help={ __( 'Anchors lets you link directly to a section on a page.' ) }
+							help={ (
+								<>
+									<p>
```

Comment:
I don't love the idea of adding extra markup in order to style the component! It's not a huge amount of work to create a stylesheet for this section (and I volunteer to do it myself if needed), is there any good reason not to though?

Reply:
Feel free to have a play and push to this branch!

Reply:
Done!